dart/lib/src/sentry.dart

```diff
@@ -33,6 +33,9 @@ class Sentry {
   /// Initializes the SDK
   /// passing a [AppRunner] callback allows to run the app within its own error
   /// zone ([`runZonedGuarded`](https://api.dart.dev/stable/2.10.4/dart-async/runZonedGuarded.html))
+  /// Starting with Flutter 3.3, the the SDK will record errors using the
+  /// [`onError`](https://api.flutter.dev/flutter/dart-ui/PlatformDispatcher/onError.html) callback
+  /// instead of running in an own error zone.
```

Comment:
I'd rephrase this, looks like it's doing both things now, or add the version constraint for the line above as well.